rcl/test/CMakeLists.txt

```diff
@@ -65,10 +65,6 @@ function(test_target_function)
 
   rcl_add_custom_gtest(test_get_node_names${target_suffix}
     SRCS rcl/test_get_node_names.cpp
-    ENV
-      ${extra_test_env}
```

Comment:
It is not strictly needed but if we register one test target per RMW_IMPLEMENTATION I think it makes sense to not run all of them with the default one